docs/src/main/asciidoc/opentracing-guide.adoc

```diff
@@ -137,3 +137,33 @@ Then visit the http://localhost:16686[Jaeger UI] to see the tracing information.
 
 
 Hit `CTRL+C` to stop the application.
+
+== Additional tracers
+
+https://github.com/opentracing-contrib[OpenTracing API Contributions] contains additional tracers that can be used to add tracing to a large variety of technologies/components.
+
+The tracer documented on the following section has been tested with Quarkus and works in both standard and native mode.
```

Comment:
```suggestion
The tracers documented in the following section has been tested with Quarkus and works in both standard and native mode.
```
(I would use the plural in the hope we would add more)

Reply:
I used plurals at first then use singular as there is only one. But I agree to use plural. Maybe I should open a followup issue to test some of the remaining tracers that can be useful ...